### PR TITLE
hlc: code cleanup

### PR DESCRIPTION
**Pull request by yaojingguo**
- fix an imprecise comment
- reuse a local variable

### CODE REVIEW

File: pkg/util/hlc/hlc.go [hunk 1]
```diff
@@ -204,10 +204,10 @@ func (c *Clock) Update(rt Timestamp) Timestamp {
 	}
 
 	// In the remaining cases, our physical clock plays no role
-	// as it is behind the local and remote wall times. Instead,
+	// as it is behind the local or remote wall times. Instead,
 	// the logical clock comes into play.
 	if rt.WallTime > c.mu.timestamp.WallTime {
-		offset := time.Duration(rt.WallTime-physicalClock) * time.Nanosecond
```
Comment (knz) [inline pkg/util/hlc/hlc.go]: I disagree with this change: the multiplication, although technically unneeded, serves as a reminder of the unit of time to the reader of this code. It is thus important for maintainability.

Comment (tamird) [inline pkg/util/hlc/hlc.go]: Yeah, let's keep it.

**Review by tamird — APPROVED** [review]

File: pkg/util/hlc/hlc.go (hunk 1 above)
Comment (yaojingguo) [inline pkg/util/hlc/hlc.go]: Yes, it has been restored in the new commit.

**Review by tamird — APPROVED** [review]
Thanks! Merge away.

**Pull request merged**